.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1 @@
+By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.
```

Comment:
Could you refer to documentation regarding the legal necessity of this message? 

Does submitting a pull request to an MIT licensed repository not already imply that the submission is made under the terms of the MIT license?